### ISSUE TITLE
feed_dict

### ISSUE THREAD

**Issue by danlutan** [OlavHN/attention-over-attention#3]
I trained the model and  test, but  the number of test sets is as large as the number of training sets.  I saw the message because the "feed _dict" was not used properly.  How to feed test set?

**Comment by cooelf** [comment]
I also encountered this problem, now I create two files for training and test respectively. But I'm not sure how this problem appears. If you have any idea, please tell me. Thx~
